containers/python/mxnet_container.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+import rpc
+import os
+import sys
+import json
+
+import numpy as np
+import cloudpickle
+import mxnet as mx
+import importlib
+
+IMPORT_ERROR_RETURN_CODE = 3
+
+MXNET_MODEL_RELATIVE_PATH = "mxnet_model"
+
+
+def load_predict_func(file_path):
+    with open(file_path, 'r') as serialized_func_file:
+        return cloudpickle.load(serialized_func_file)
+
+
+# load mxnet model from serialized dir
+def load_mxnet_model(model_path):
+    data_iter = mx.io.CSVIter(
+        data_csv="train.data", data_shape=(785, ), batch_size=1)
+
+    sym, arg_params, aux_params = mx.model.load_checkpoint(
+        prefix=model_path, epoch=0)
+    mxnet_model = mx.mod.Module(symbol=sym)
+    mxnet_model.fit(
```

Comment:
The user's model should be fit (trained) prior to deployment. We should not be calling any additional training functions within the container. Rather, the model should be loaded from its saved checkpoint files (symbol and parameter). Let me know if you have any questions.